segmenters/image/FaceNetSegmenter/requirements.txt

```diff
@@ -1,2 +1,3 @@
 facenet-pytorch==2.5.2
 torchvision==0.9.0
+opencv-python
```

Comment:
fix the version

Reply:
the issue was need to install opencv-python's dependency libGL in dockerfile

Reply:
and I will specify the opencv-python version

Reply:
remember this

Reply:
oh yes